osconfig_tests/test_suites/patch/test_setup.go

```diff
@@ -61,6 +61,24 @@ if (Test-Connection $wu_server -Count 1 -ErrorAction SilentlyContinue) {
 	Set-ItemProperty -Path $windows_update_path -Name WUStatusServer -Value "http://${wu_server}:8530"
 	Set-ItemProperty -Path $windows_update_au_path -Name UseWUServer -Value 1
 }
+`
+
+	windowsLocalPostPatchScript = `
+New-Item -Path . -Name "windows_local_post_patch_script.cmd" -ItemType "file" -Value "echo OS Config - local patch script - Windows > post_patch_script_out.txt"
+`
+
+	windowsPrePostStepValidation = `
```

Comment:
Can we make the scripts themselves write to guest attributes, that way we don't need to have this validation run at all

Reply:
Thanks, it made the setup much simpler, I tested the new scripts on the same images as mentioned in the top comment, but not on windows because of bug 141310538